### PR TITLE
magento/devdocs#7366: GraphQL: Document password change mutations. Mutation: requestPasswordResetEmail.

### PR DESCRIPTION
**Pull request by atwixfirster**
## Purpose of this pull request

This pull request (PR) adds a new topic about `requestPasswordResetEmail` mutation in scope of https://github.com/magento/devdocs/issues/7364.

## Affected DevDocs pages

- https://devdocs.magento.com/guides/v2.4/graphql/mutations/revoke-customer-token.html

## Links to Magento source code

- https://github.com/magento/magento2/pull/27876

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/guides/v2.4/graphql/mutations/request-password-reset-email.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+---
+group: graphql
+title: requestPasswordResetEmail mutation
+contributor_name: Atwix
+contributor_link: https://www.atwix.com/
+---
+
+The `requestPasswordResetEmail` mutation triggers the password reset email by the provided email address. Use it to initiate a reset password process for the registered customer before [resetPassword]({{page.baseurl}}/graphql/mutations/reset-password.html) mutation call.
```
Comment (rogyar) [inline src/guides/v2.4/graphql/mutations/request-password-reset-email.md:8]: Could you double-check, please, that we have the "Reset Password" related page? I don't see it neither in the mainline nor in the 2.4.1 branch. 

We have another page for changing customer password (https://devdocs.magento.com/guides/v2.4/graphql/mutations/change-customer-password.html) but I may be missing something. 

Thank you

Comment (atwixfirster) [reply src/guides/v2.4/graphql/mutations/request-password-reset-email.md:8]: Yes.

Check this one - https://github.com/magento/devdocs/pull/7426

THank you, @rogyar

**Comment by atwixfirster** [comment]
> Approved. If you haven't optimized the PNG file already, please run the `rake check:image_optim path=src/guides/v2.4/graphql/images/reset-password-email.png` from the devdocs home directory.

**done**

**Comment by keharper** [comment]
running tests

**Pull request merged**